src/utils/locale.ts

```diff
@@ -0,0 +1,12 @@
+export type Locales = 'en' | 'it';
+
+// return the current locale set in the device (this could be different from the app supported languages)
+// export const getCurrentLocale = (): Locales => I18n.currentLocale();
+
+/**
+ * return the primary component of the current locale (i.e: it-US -> it)
+ * if the current locale (the language set in the device) is not a language supported by the app
+ * the fallback will returned
+ * @param fallback
+ */
+export const getLocalePrimaryWithFallback = (fallback: Locales = 'it') => fallback;
```

Comment:
Not sure if this is relevant here as this was made for react native's l12n apis

Reply:
I imported `input.ts` as it is a dependency of `types/pagopa.ts`. Maybe I could purge it from useless definitions. `getLocalePrimaryWithFallback` is used in `PaymentManagerClient`, so I left it.